test/api/v3/unit/models/group.test.js

```diff
@@ -607,6 +613,90 @@ describe('Group Model', () => {
       });
     });
 
+    describe('#checkChatSpam', () => {
+      let testMessage, testUser, testTime;
+      let testUserID = '1';
+      beforeEach(async () => {
+        party._id = TAVERN_ID;
+        testTime = Date.now();
+        testMessage = {
+          text: 'test message',
+          uuid: testUserID,
+          timestamp: testTime,
+        };
+        testUser = {
+          _id: testUserID,
+        };
+      });
+
+      it('group that is not the tavern returns false', async () => {
+        party._id = '2';
```

Comment:
For better coverage, this should include a condition that if it _was_ the Tavern, it would return true.